src/Microsoft.DocAsCode.Plugins/ISupportIncrementalPostProcessor.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Plugins
+{
+    public interface ISupportIncrementalPostProcessor
+    {
+        /// <summary>
+        /// Get post processor host
+        /// </summary>
+        IPostProcessorHost PostProcessorHost { get; set; }
```

Comment:
in what scenario, do we need to set host in "outer code"?

---
In reply to: [94198112](https://github.com/dotnet/docfx/pull/1081#discussion_r94198112) [](ancestors = 94198112,94111780)